deploy/helm/sumologic/templates/prometheus-rules.yaml

```diff
@@ -0,0 +1,189 @@
+{{- $kubeTargetVersion := default .Capabilities.KubeVersion.GitVersion .Values.kubeTargetVersionOverride }}
+{{- if semverCompare ">=1.14.0-0" $kubeTargetVersion }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
```

Comment:
are we not supporting yaml based install for these? (this template is Helm-only).

Reply:
ahh good point, let me add that.

Reply:
actually I might be confused, I see it's added as a section to the generated `fluentd-sumologic.yaml.tmpl`